### PR TITLE
Update blog tutorial

### PR DESCRIPTION
**Pull request by Tosch110**
Update blog to Stargate

### CODE REVIEW

File: blog/blog/x/blog/module.go [hunk 1]
```diff
@@ -1,147 +1,167 @@
...
-	return cli.GetTxCmd(cdc)
+// GetTxCmd returns the capability module's root tx command.
+func (a AppModuleBasic) GetTxCmd() *cobra.Command {
+	return cli.GetTxCmd()
 }
 
-// GetQueryCmd returns no root query command for the blog module.
-func (AppModuleBasic) GetQueryCmd(cdc *codec.Codec) *cobra.Command {
-	return cli.GetQueryCmd(types.StoreKey, cdc)
+// GetQueryCmd returns the capability module's root query command.
```
Comment (marbar3778) [inline blog/blog/x/blog/module.go:92]:
```suggestion
// GetQueryCmd returns the blog's module root query command.
```
?

File: blog/blog/x/blog/module.go [hunk 2]
```diff
@@ -1,147 +1,167 @@
...
-	var data types.GenesisState
-	err := types.ModuleCdc.UnmarshalJSON(bz, &data)
-	if err != nil {
-		return err
+// RegisterInterfaces registers the module's interface types
+func (a AppModuleBasic) RegisterInterfaces(reg cdctypes.InterfaceRegistry) {
+	types.RegisterInterfaces(reg)
+}
+
+// DefaultGenesis returns the capability module's default genesis state.
```
Comment (marbar3778) [inline blog/blog/x/blog/module.go:63]:
```suggestion
// DefaultGenesis returns the blog module's default genesis state.
```

File: blog/blog/x/blog/genesis.go [hunk 3]
```diff
@@ -4,19 +4,14 @@ import (
...
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 }
 
-// ExportGenesis writes the current store values
-// to a genesis file, which can be imported again
-// with InitGenesis
-func ExportGenesis(ctx sdk.Context, k keeper.Keeper) (data types.GenesisState) {
-	// TODO: Define logic for exporting state
-	return types.NewGenesisState()
+// ExportGenesis returns the capability module's exported genesis.
```
Comment (marbar3778) [inline blog/blog/x/blog/genesis.go:14]: Is starport code gen taken from the capability module?

Comment (Tosch110) [reply blog/blog/x/blog/genesis.go:14]: That's a question for starport scaffold @ilgooz

Comment (ilgooz) [reply blog/blog/x/blog/genesis.go:14]: Not necessarily the cap module. We take sdk/simapp, sdk/x, gaia, akash and akash/x as a reference for different parts of the scaffold.

Comment (ilgooz) [reply blog/blog/x/blog/genesis.go:14]: Yep, definitelly! We need to review all the comments in source code and fix similar issues. Tracked by https://github.com/tendermint/starport/issues/665.

Comment (Tosch110) [reply blog/blog/x/blog/genesis.go:14]: Thanks, something for starport to look into

**Review by lukitsbrian — APPROVED** [review]

**Pull request merged**